### PR TITLE
feat: button to load liquidation rewards

### PR DESCRIPTION
**Pull request by nieldeckx**
Screenshot 2022-05-18 at 13 36 53

Screenshot 2022-05-18 at 13 37 15

### CODE REVIEW

File: web/components/liquidations.tsx [hunk 1]
```diff
@@ -514,16 +517,35 @@ export const Liquidations: React.FC = () => {
...
+                      </div>
                     </div>
-                      <div className="w-full bg-gray-200 rounded-full h-2.5 dark:bg-gray-700">
-                      <div className="bg-indigo-600 h-2.5 rounded-full" style={{ width: rewardLoadingPercentage + "%" }}></div>
+                  </div>
+                ) : isLoadingRewards ? (
+                  <div className="mt-4 shadow sm:rounded-md sm:overflow-hidden">
+                    <div className="px-4 py-5 bg-white dark:bg-zinc-800 sm:p-6">
+                      <div className="flex justify-between mb-3">
+                        <span className="text-base font-medium dark:text-white">Checking liquidated vaults..</span>
```
Comment (Macxim) [inline web/components/liquidations.tsx:541]:
```suggestion
                        <span className="text-base font-medium dark:text-white">Checking liquidated vaults…</span>
```

**Review by Macxim — APPROVED** [review]

**Comment by Macxim** [comment]
Fantastic  ✨

**Comment by philipdesmedt** [comment]
@nieldeckx @Macxim should we make the copy at the end a bit less paternalistic?

Maybe something like "Thanks for your patience!"

**Comment by Macxim** [comment]
Agree! Occupational hazard maybe @nieldeckx? 😅

**Pull request merged**